sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java

```diff
@@ -17,25 +18,58 @@
 import io.opentelemetry.sdk.autoconfigure.spi.ConfigurableSpanExporterProvider;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
 import java.time.Duration;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
 import java.util.ServiceLoader;
-import java.util.stream.Collectors;
+import java.util.Set;
+import java.util.function.Function;
 import java.util.stream.StreamSupport;
-import javax.annotation.Nullable;
 
 final class SpanExporterConfiguration {
 
-  @Nullable
-  static SpanExporter configureExporter(String name, ConfigProperties config) {
+  private static final String EXPORTER_NONE = "none";
+
+  // Visible for testing
+  static Map<String, SpanExporter> configureSpanExporters(ConfigProperties config) {
+    List<String> exporterNamesList = config.getCommaSeparatedValues("otel.traces.exporter");
+    Set<String> exporterNames = new HashSet<>(exporterNamesList);
+    if (exporterNamesList.size() != exporterNames.size()) {
+      throw new ConfigurationException("otel.traces.exporter contains duplicates");
```

Comment:
NIT: we could name the duplicates in the exception message (or log). In case of long exporter lists this may be helpful.